### PR TITLE
Repo update

### PR DESCRIPTION
**Pull request by YohanesGetinet1**
I updated some files in the repo and added some new lists
Fixed some bugs in the README file

### CODE REVIEW

File: Cloud_providers.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	<item>Microsoft Azure</item>
+	<item>Google Cloud </item>
+	<item>IBM Cloud</item>
+	<item>Oracle Cloud</item>
+	<item>Alibaba Cloud</item>
+	<item>RedHat</item>
+	<item>Heroku</item>
+	<item>Digital Ocean</item>
+	<item>CloudFlare</item>
+    <item>Linode</item>
```
Comment (vinaygaba) [inline Cloud_providers.xml]: indentation here is inconsistent

File: App Categories.xml [hunk 2]
```diff
@@ -1,3 +1,53 @@
 <string-array name="app_categories">
+    <item>Art &amp; Design</item>
```
Comment (vinaygaba) [inline App Categories.xml:2]: does this map 1-1 with the categories in the Google Play store?

File: Countries.xml [hunk 3]
```diff
@@ -67,6 +67,7 @@
     <item>Equatorial Guinea</item>
     <item>Eritrea</item>
     <item>Estonia</item>
+    <item>Eswatini</item>
```
Comment (vinaygaba) [inline Countries.xml:70]: 👍

Comment (vinaygaba) [inline Countries.xml:70]: 👍

File: Genre.xml [hunk 4]
```diff
@@ -214,7 +214,7 @@
 	<item>Contemporary Instrumental</item>
 	<item>Contemporary Jazz</item>
 	<item>Contemporary Native American</item>
-	<item>Contemporary R&B</item>
+	<item>Contemporary R&amp;B</item>
```
Comment (vinaygaba) [inline Genre.xml:217]: good idea to use `&amp;`

File: README.md [hunk 5]
```diff
@@ -5,27 +5,32 @@ Ultimate String-Array List
...
 
 Current List of String-Arrays
 -----------------------------
+
 The repository has the following list of string-array. Please note that this is continuously being updated so do check frequently. You can also send in your requests via the issue tracker. I am hoping that you take some inspiration from the categories I am adding and make contributions to this repo to make it more useful.
 
 Name|Description
 ----|-----------
 [App Categories.xml](/App%20Categories.xml)| List of app categories present on the Google Play Store
+[aws_regions.xml](/aws_regions.xml)| List of codes and names for AWS region
```
Comment (vinaygaba) [inline README.md]: Rename this to `AWS Regions.xml` to have consistent naming

File: world_cup_2022_groups.xml [hunk 6]
```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (vinaygaba) [inline world_cup_2022_groups.xml:1]: Nice! Great addition!

**Review by vinaygaba — APPROVED** [review]

File: Cloud_providers.xml (hunk 1 above)
Comment (YohanesGetinet1) [reply Cloud_providers.xml]: Sorry , I will fix it

File: App Categories.xml (hunk 2 above)
Comment (YohanesGetinet1) [reply App Categories.xml:2]: Sorry, I didn't get you meant whether the Art and Design category exists or not.

Comment (vinaygaba) [reply App Categories.xml:2]: @YohanesGetinet1 no I was talking about all the categories in general. Are these the exact same categories that you see for the apps on the Google Play Store?

Comment (YohanesGetinet1) [reply App Categories.xml:2]: Ok, @vinaygaba thanks for the clarity 
most of the categories are in Google Play Store but some of the categories are sub-categories for the main ones. Should I have to include the main categories only?

**Pull request merged**